packages/storefront-sdk-plugins/commerce-queries/src/graphql/queries/content.ts

```diff
@@ -0,0 +1,18 @@
+import fragments from '../fragments/index.js';
+export default /* GraphQL */ `
+	query allContent($filter: ContentFilterInput) {
+		allContent(filter: $filter) {
+			pageInfo {
+				hasNextPage
+				endCursor
+			}
+			edges {
+				cursor
+				node {
+					...content_content
```

Comment:
Recommend sticking to the convention of the prefix being the PascalCase GraphQL type:

```suggestion
					...Content_content
```